gazebo/rendering/LensFlare.cc

```diff
@@ -268,37 +271,27 @@ namespace gazebo
       lightPos.z = (imagePos.Z() > 1.75) ? -1 : 1;
 
       // check occlusion and set scale
-      // loop through all env cameras and find the cam that sees the light
-      // ray cast using that env camera to see if the distance to closest
-      // intersection point is less than light's world pos
       double occlusionScale = 1.0;
       if (lightPos.z >= 0.0)
       {
-        // loop through all env cameras
-        for (auto cam : ogreEnvCameras)
-        {
           // project light world point to camera clip space.
           auto viewProj = cam->getProjectionMatrix() * cam->getViewMatrix();
```

Comment:
replace `cam` with `_wideAngleCam` two times on this line

Reply:
Hmm, ``cam`` doesn't have ``getProjectionMatrix()`` method here.